hadoop-project/pom.xml

```diff
@@ -1288,10 +1288,22 @@
         <artifactId>jackson-dataformat-cbor</artifactId>
         <version>${jackson2.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-bom</artifactId>
+        <version>${mockito.version}</version>
+        <type>pom</type>
```

Comment:
This would import the dependency-management of mockito-bom. From, https://central.sonatype.com/artifact/org.mockito/mockito-bom, looks like it also has mockito-android. Was curious that if the dll files are coming because of this. Also, do we need the bom, since we are adding dependencies for both mockito-core and mockito-inline (because we would be importing many other dependencies which might not be of use).

Reply:
be safer to just pull in the ones we know we need

Reply:
I agree with the decision to remove the 'mockito-bom' dependency. However, I've encountered an issue where the DLL files are not being included due to the removal of this dependency. The DLL files are originally included as part of the 'byte-buddy-agent' POM, and 'mockito-core' has a dependency on this module. Despite setting the value of 'mockito-core' as optional in 'hadoop-minicluster,' the transitive dependency is still affecting 'hadoop-client-check-test-invariants,' resulting in an error during the check for jar contents due to disallowed DLL values. Consequently, I've addressed this by adding the necessary expressions to allow these DLL files